lib/closure-linter-wrapper.js

```diff
@@ -140,8 +140,8 @@ function execute(program, options, callback) {
     var flags = options.flags || [],
         src = options.src || [],
         params = [],
-        cmd = path.normalize('python ' + __dirname +
-            '/../tools/' + program + '.py'),
+        cmd = path.normalize('python "' + __dirname +
```

Comment:
@dcantelar maybe it's better to escape single quotes so linting doesn't fail:

`cmd = path.normalize('python \'' + __dirname +
    `/../tools/' + program + '-py\''),